src/useDebouncedCallback/index.ts

```diff
@@ -0,0 +1,25 @@
+import { useCallback, useEffect, useRef } from "react"
+
+/**
+ * debouncedCB hook.
+ */
+export function useDebouncedCallback(callback: (...args: any[]) => void, delay: any, deps: React.DependencyList) {
```

Comment:
You could destructure the import from `"react"` here:

```suggestion
export function useDebouncedCallback(callback: (...args: any[]) => void, delay: any, deps: DependencyList) {
```

since you're already using other named imports on the top.

Reply:
👍